golos.publication/parameters.hpp

```diff
@@ -58,10 +59,24 @@ namespace golos {
         }
     };
     using referral_acc_prm = param_wrapper<st_referral_acc, 1>;
+    
+    struct st_curators_prcnt : parameter {
+        uint16_t min_curators_prcnt;
+        uint16_t max_curators_prcnt;
+
+        void validate() const override {
+            eosio_assert(min_curators_prcnt >= 0 && min_curators_prcnt <= config::_100percent,
+                    "Min curators percent must be greater than 0 and less than 100 or equal.");
```

Comment:
`"must be greater than 0"` is wrong, 0 is acceptable too. maybe `"must be between 0% and 100% (0-10000)"`